source/includes/fact-connect-intro.rst

```diff
@@ -1,7 +1,9 @@
 The :ref:`mongosync <c2c-mongosync>` utility creates a connection
 between MongoDB clusters. ``mongosync`` can make a connection between:
 
-- :ref:`self-managed clusters <server-replica-set-deploy-all>`,
-- :atlas:`MongoDB Atlas </>` hosted clusters, or 
-- a self-managed cluster and an Atlas hosted cluster.
-
+- :ref:`self-managed clusters <server-replica-set-deploy-all>`
+- :atlas:`MongoDB Atlas </>` hosted clusters
+- a self-managed cluster and an Atlas hosted cluster
+- :ref:`clusters that support Workforce Identity Federation
```

Comment:
This seems like a strange place to list an authentication mechanism.

Atlas clusters are deployed to Azure, GCP, or Amazon. ([link](https://www.mongodb.com/docs/atlas/reference/cloud-providers/)) So the above lines that talk about “Atlas hosted cluster” already implicitly encompass the special case of Azure- or GCP-deployed clusters that authenticate via OIDC.

It seems, then, like this file should remain unchanged. @pmeredit, thoughts?

Reply:
I agree, we don't need to add this here.

Reply:
Ok, removed.